src/patches/HEAD/0001-Fix-UBSAN-errors-in-decompiler.patch

```diff
@@ -83,7 +83,7 @@ index aebcfd910..cf079f20e 100644
  {
 -  uintb res = (in1>>(in2*8)) & calc_mask(sizeout);
 +  uintb res = in1;
-+  if ((in2*8) < sizeof(in1)) {
++  if (in2 < sizeof(in1)) {
```

Comment:
This issue isn't new. Upstream has recently added some more test coverage so this bug in the patch is only now being triggered.

Reply:
Ah! Is there an upstream PR that needs to be updated?

Reply:
Also, does this fix need to be backported to the stable patch?

Reply:
The upstream PR has a few merge conflicts. I've just left a comment saying I can resolve conflicts if they have any interest in getting it in. But I won't continue keeping it up to date unless they respond since I opened it ~8 months ago.

Regarding the stable patch, that's a good point. I'll do that now.

Reply:
[5a50dac](https://github.com/lifting-bits/sleigh/pull/195/commits/5a50dac49f89fe484c991a7fc3927f689f7b2953)

Updated stable patch.